ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -115,7 +115,7 @@ struct
     pbis_config "CreateHomeDir" "false";
     pbis_config "SyncSystemTime" "false";
     pbis_config "LdapSignAndSeal" "true";
-    pbis_config "CacheEntryExpiry" "300";
+    pbis_config "CacheEntryExpiry" "0";
```

Comment:
If we don't do this. The CA issue will still exist, since a cache data sometimes is also a wrong data. Moreover, the pbis command is not frequently being invoked. It seems there is no much need to have a cache here.